addons/web/static/src/views/fields/monetary/monetary_field.xml

```diff
@@ -3,10 +3,10 @@
 
     <t t-name="web.MonetaryField">
         <span t-if="props.readonly" t-esc="formattedValue" />
-        <div class="text-nowrap d-inline-flex w-100 align-items-baseline position-relative" t-else="">
-            <span class="o_input position-absolute pe-none d-flex w-100">
+        <div class="text-nowrap d-inline-flex align-items-baseline position-relative" t-else="">
+            <span class="o_input position-absolute pe-none d-flex">
                 <span t-if="!props.hideSymbol and currency and currency.position === 'before'" t-out="currencySymbol + nbsp"/>
-                <span class="opacity-0 d-inline-block overflow-hidden mw-100 o_monetary_ghost_value" t-out="state.value"/>
+                <span class="opacity-0 d-inline-block overflow-hidden o_monetary_ghost_value" t-out="state.value"/>
```

Comment:
this change doesn't seem to change anything on 2-3 fields I checked, but it seems the most risky change since it affects any Monetary field.

If it's really useless I guess it's ok. Otherwise we should rather have a fix in the product_pricelist_item_form_view than here.

Reply:
in mobile, it changes that a monetary field doesn't take the full row

before the change in monetary_field.xml:

> ![2024-08-28-154848_341x138_scrot](https://github.com/user-attachments/assets/c9e2024e-b885-4f2f-8830-642112c785f4)

after the change:
> ![2024-08-28-154838_342x115_scrot](https://github.com/user-attachments/assets/e1937e15-d2d6-4fcf-9a53-58d4b81c0331)

it does the same in desktop, and it also brings an issue where if we put a very long number in the field, the "o_monetary_ghost_value" span will burst out of the field outside of the child but not content is shown (:

> ![2024-08-28-155225_896x39_scrot](https://github.com/user-attachments/assets/ffd9d096-e649-4f9d-b526-c20afb8e430c)

So we probably shouldn't change the monetary field.

Reply:
makes sense, i updated it to just increase a bit the size of the field.